internal/declarative/v2/inmemory_rendered.go

```diff
@@ -30,10 +34,8 @@ type InMemoryManifestCache struct {
 func (c *InMemoryManifestCache) Parse(
 	ctx context.Context, renderer Renderer, obj Object, spec *Spec,
 ) (*internal.ManifestResources, error) {
-	file := filepath.Join(manifest, spec.Path, spec.ManifestName)
-	hashedValues, _ := internal.CalculateHash(spec.Values)
```

Comment:
This is removed, and from what I can see, the `spec.Values` are no longer used. Perhaps it should be alse removed then?

Reply:
which should be removed? the `spec.Values` previously used as part of the cache key, current change is just avoid using it as the cache key, but the cache is still valid

Reply:
@ruanxin I meant `Spec.Values` from `internal/declarative/v2` package. After your changes I see it's no longer used in the project. Or I missed something? If the `Spec.Values` is no longer used, maybe it's good time to also remove it from the project?

Reply:
good catch, `Spec.Values` removed